tests/multidevice/multidevice.cpp

```diff
@@ -63,9 +60,25 @@ class multidevice_sync_ft : public ::testing::Test
     virtual void SetUp()
     {
         srand((unsigned int)time(0)); // use current time as seed for random generator
+        ASSERT_EQ(m_master, nullptr);
+        ASSERT_EQ(m_subordinate, nullptr);
     }
 
-    virtual void TearDown() {}
+    virtual void TearDown()
+    {
+        if (m_master != nullptr)
+        {
+            k4a_device_close(m_master);
+            m_master = nullptr;
+        }
+        if (m_subordinate != nullptr)
+        {
+            k4a_device_close(m_subordinate);
+            m_subordinate = nullptr;
+        }
+    }
+    k4a_device_t m_master = nullptr;
+    k4a_device_t m_subordinate = nullptr;
```

Comment:
I don't think we want to add the types here to `m_master` and `m_subordinate`. This is just declaring unused local variables right?

Reply:
No, this is the class that gets constructed first, and setup and teardown are calls that run at the beginning and end of the program. Tear down runs in the event of a crash so it can help ensure cleanup.

Reply:
Let me ask a different way, do we need to have the type of `m_master` and `m_subordinate` or could we just re-write these two lines as (notice the removal of `k4a_device_t` from each line):

```c++
m_master = nullptr;
m_subordinate = nullptr;
```

I _think_ (but am not sure) that adding the `k4a_device_t` in front of `m_master` and `m_subordinate` has made it so that we are not referring to local variables instead of member variables.

Reply:
Wes talked  with me about this, I thought this was still in a function defintion, but it's obviously in a class definition which makes more sense.